Makefile: do not execute the rule "all" when executing "install"

signed-off-by: Arnaud Aujon <arnaud.aujon@...>
===================================================================
--- espeak-1.47.11-source.orig/src/Makefile
+++ espeak-1.47.11-source/src/Makefile
@@ -131,7 +131,7 @@
	rm -f $(BIN2_NAME)
	rm -f $(LIB_NAME)*

-install: all
+install:
	# Create directories
	rm -rf $(DESTDIR)$(DATADIR)
	$(MKDIR) $(DESTDIR)$(BINDIR)
